Makefile

```diff
@@ -373,7 +380,7 @@ protoc-gen-go-grpc: ## Download protoc-gen-go-grpc locally if necessary.
 e2e-test:
 	# KUBECONFIG must be set to the cluster, and PP needs to be deployed already
     # count arg makes the test ignoring cached test results
-	go test ./e2e -ginkgo.v -ginkgo.progress -test.v -timeout 60m -count=1 ${TEST_OPS}
+	go test ./e2e -ginkgo.v -ginkgo.show-node-events -test.v -timeout 60m -count=1 ${TEST_OPS}
```

Comment:
This is nice, it would be interesting to see the change of verbosity on failure runs!
https://onsi.github.io/ginkgo/#controlling-verbosity

> Very-verbose mode contains additional information over verbose mode. In particular, -vv timelines indicate when individual nodes start and end and also include the full failure descriptions for every failure encountered by the spec. Verbose mode does not include the node start/end events (though this can be turned on with --show-node-events) and does not include detailed failure information for anything other than the first (primary) failure. (Additional/subseuqent failures typically occur in clean-up nodes and are not as relevant as the primary failure that occurs in a subject or setup node).

It would have been nice to leverage that in unit testing as well at `test` target but it [doesn't use ginkgo](https://github.com/medik8s/self-node-remediation/pull/206/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R187)

Reply:
> target but it doesn't use ginkgo

I don't think it's a problem, `go test` supports passing argument to ginkgo via `-gingko.<flag>` syntax

Reply:
I'll try with `-vv` and see if it is enough alone, or it still needs `--show-node-events`

Reply:
`-vv` is equal to `-v` + `--show-node-events` + `detailed failure information`
This means that is quite valuable in e2e tests when run in CI.
I'm not completely sure it's as good as in unittests. We (can) also run them locally and in that context I lean over a short report, since I can always re-run with more specific option defining `TEST_OPTS` env variable

Reply:
SGTM